src/kem/sike/CMakeLists.txt

```diff
@@ -23,22 +23,27 @@ if(WIN32 AND NOT (MINGW OR MSYS OR CYGWIN))
     add_compile_options(/wd4133)
 endif()
 
-set(SRCS kem_sike.c
-         P434/P434.c
+set(SRCS P434/P434.c
          P434/P434_compressed.c
          P503/P503.c P503/P503_compressed.c
          P610/P610.c
          P610/P610_compressed.c
          P751/P751.c
          P751/P751_compressed.c)
+if(OQS_ENABLE_KEM_SIKE)
+    set(SRCS ${SRCS} kem_sike.c)
+endif()
+if(OQS_ENABLE_KEM_SIDH)
+    set(SRCS ${SRCS} kem_sidh.c)
+endif()
 
 if(ARCH STREQUAL "x86_64")
     if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Linux")
         set(SRCS ${SRCS} P503/AMD64/fp_x64_asm.S
                          P751/AMD64/fp_x64_asm.S)
-        if(USE_BMI2_INSTRUCTIONS AND NOT OQS_PORTABLE_BUILD)
+        if(OQS_USE_BMI2_INSTRUCTIONS AND NOT OQS_PORTABLE_BUILD)
```

Comment:
I opened another PR with this particular fix, since it would be worth having for the release.

Reply:
Everything's fine. I removed that code from my PR